tests/stdlib/talgorithm.nim

```diff
@@ -113,3 +113,82 @@ block:
     doAssert binarySearch(moreData, 6) == -1
     doAssert binarySearch(moreData, 4711) == 4
     doAssert binarySearch(moreData, 4712) == -1
+
+# merge
+block:
+  var x = @[1, 7, 8, 11, 21, 33, 45, 99]
+  var y = @[6, 7, 9, 12, 57, 66]
+
+  let merged = merge(x, y)
+  doAssert merged.issorted
+  doAssert merged == @[1, 6, 7, 7, 8, 9, 11, 12, 21, 33, 45, 57, 66, 99]
+
+block:
+  var x = @[111, 88, 76, 56, 45, 31, 22, 19, 11, 3]
+  var y = @[99, 85, 83, 82, 69, 64, 48, 42, 33, 31, 26, 13]
+
+  let merged = merge(x, y, SortOrder.Descending)
+  doAssert merged.issorted(SortOrder.Descending)
```

Comment:
```suggestion
  doAssert merged.isSorted(SortOrder.Descending)
```

Reply:
@ee7 no need to repeat the same feedback for each instance (same as https://github.com/nim-lang/Nim/pull/16182#discussion_r532391892)